src/index.html

```diff
@@ -4,9 +4,342 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img
+          src="/src/images/logo.png"
+          alt="Moyo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="index.html" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
```

Comment:
```suggestion
          </li>
          
          <li class="nav__item">
```

Keep same code style everywhere